tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1310,14 +1309,14 @@ exports[`no_code.php 1`] = `
 ?>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <?php
-/**
+
+?>/**
```

Comment:
Regression?

Reply:
@czosel no, it is bug not related to inline (look on preview print, he is also buggy), we should fix it in `comments.js`